example/tailor.js

```diff
@@ -1,12 +1,16 @@
 'use strict';
 const http = require('http');
 const path = require('path');
+const fs = require('fs');
 const Tailor = require('../index');
 const fetchTemplateFs = require('../lib/fetch-template');
 const serveFragment = require('./fragment');
 const baseTemplateFn = () => 'base-template';
-
+// You can also pass AMD URL
+// const AMD_LOADER = 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.22/require.min.js';
+const AMD_LOADER = fs.readFileSync(require.resolve('iamdee'), 'utf-8');
```

Comment:
To be honest I would accept `file://` urls and still use require.js by default since it's more battle-proven